src/types.ts

```diff
@@ -0,0 +1,29 @@
+// contains all the types that we want to use
```

Comment:
All types across the whole of the codebase?

Reply:
these are custom types - I'll update the comment.